frontend/src/main/components/Nav/AppNavbar.js

```diff
@@ -63,6 +63,11 @@ export default function AppNavbar({
                   Personal Schedules
                 </Nav.Link>
               )}
+              {hasRole(currentUser, "ROLE_USER") && (
+                <Nav.Link href="/" data-testid="appnavbar-mainsearch-list">
+                  Main Search
+                </Nav.Link>
+              )}
```

Comment:
Why are we gating this on ROLE_USER?

That requires folks to be logged in.  I don't think we've ever wanted folks to have to be logged in to do simple searchers for courses.  They only need to be logged in if they are building personal schedules or doing admin features.